cea/demand/thermal_loads.py

```diff
@@ -81,37 +79,33 @@ def calc_thermal_loads(building_name, bpr, weather_data, usage_schedules, date,
         ventilation_air_flows_simple.calc_m_ve_leakage_simple(bpr, tsd, gv)
 
         # get internal comfort properties
-        tsd = controllers.calc_simple_temp_control(tsd, bpr.comfort, gv.seasonhours[0] + 1, gv.seasonhours[1],
-                                                   date.dayofweek)
-
-
+        tsd = control_heating_cooling_systems.calc_simple_temp_control(tsd, bpr, date.dayofweek)
 
         # end-use demand calculation
-        for t in range(-720, 8760):
-            hoy = helpers.seasonhour_2_hoy(t, gv)
+        for t in get_hours(bpr):
```

Comment:
nice!! (`get_hours` could be a method of `bpr` i guess?)

Reply:
If you think so.